demo/babel.config.js

```diff
@@ -2,6 +2,9 @@ module.exports = function(api) {
   api.cache(true);
   return {
     presets: ["babel-preset-expo"],
-    plugins: ["transform-remove-console"],
+
+    // Note: console.* should be removed from production app because performance issues
+    // See more: https://docs.expo.io/versions/latest/react-native/performance/#using-consolelog-statements
+    //plugins: ["transform-remove-console"],
```

Comment:
Why comment it out?

Reply:
Because it's silent expo warnings and useful debug console.log too.
I think that with `ethers.js` loglevel last change the performance issue was addressed.

Reply:
But I thought it only happened in production? So you would still see useful debug console.log while developing locally, right?

Reply:
Right

Reply:
The "definitive" solution here is use the plugin only on prod build, I've tryied for a while to work with babel/expo `env` params but has some workarounds to do so I thought better keep that way for know and improve build scripts later.

Reply:
So then maybe we should uncomment this?

Reply:
I made my comment above just before you made your last one.

Okay, I assumed the plugin worked only in prod by default or something, so I was wrong about that.

Makes sense and sounds good! We can always work on this again later when it comes up again.

Reply:
Resolving this